django/test/runner.py

```diff
@@ -1179,7 +1180,7 @@ def reorder_tests(tests, classes, reverse=False, shuffler=None):
         yield from reorder_test_bin(tests, shuffler=shuffler, reverse=reverse)
 
 
-def partition_suite_by_case(suite):
+def partition_suite_by_test_class(suite):
```

Comment:
Please revert this change. PR should be limited to docs changes.

Reply:
Reverted